### PR TITLE
adding focus and keyboard accessibility to the cookie consent buttons

### PR DESCRIPTION
**Pull request by Amberroseweeks**
- You can now tab to the cookie options when they appear and when you select the button in the footer
- You can activate the cookie settings by pressing ENTER or SPACEBAR on the button in the footer
- The cookie setting are working for me on Apple VO, I have applied aria-describedby on the div and added the role of region. 

Note: I added positive tab values for the Accept and Decline buttons. This feels a little wrong, I am not sure if this is best practice, but it did seem to work in allowing those items to receive focus first.

This closes issue https://github.com/CodeForPhilly/clean-and-green-philly/issues/859

### CODE REVIEW

File: src/components/CookieConsentBanner.tsx [hunk 1]
```diff
@@ -21,34 +24,54 @@ const CookieConsentBanner = () => {
...
           <ThemeButton
+            tabIndex={1}
             color="tertiary"
             label="Decline"
             startContent={<X />}
             onPress={() => onClickButton(false)}
           />
 
           <ThemeButton
+            tabIndex={2}
```
Comment (bacitracin) [inline src/components/CookieConsentBanner.tsx:74]: This is one of those exceptions to the rule that I think is okay. 
Although I have seen it said somewhere on the A11y Slack channel that by now screenreader users just expect that the cookie banner will be on the bottom and they will TAB backwards to get to it.

File: src/components/CookieConsentBanner.tsx [hunk 2]
```diff
@@ -3,12 +3,15 @@
 import { ThemeButton } from "./ThemeButton";
 import { Check, X } from "@phosphor-icons/react";
 import { useCookieContext } from "@/context/CookieContext";
-import { useEffect, useState } from "react";
+import { useEffect, useState, useRef } from "react";
```
Comment (bacitracin) [inline src/components/CookieConsentBanner.tsx:6]: It looks like you have a couple of small conflicts that need to be resolved with main.

**Review by bacitracin — APPROVED** [review]
Nice work! You just need to fix your conflicts

**Comment by CodeWritingCow** [comment]
Hi @Amberroseweeks I've changed the PR target branch from `main` to `staging`. In the future, let's submit PRs against `main` branch. Thanks!

**Comment by CodeWritingCow** [comment]
Hi @Amberroseweeks, looks like the latest Vercel deployment to its preview env has failed. Can you run lint in your local env and see if any lint error appears? Thanks!

**Comment by bacitracin** [comment]
Hey @Amberroseweeks Just checking in on this PR! Can you please check for failing tests or lint errors locally?

**Comment by Amberroseweeks** [comment]
> Hey @Amberroseweeks Just checking in on this PR! Can you please check for failing tests or lint errors locally?

I think I have successfully fixed the issue, I pulled changes from main the combines my changes.

**Pull request merged**